core/src/main/java/tc/oc/pgm/rotation/VotingPool.java

```diff
@@ -88,10 +95,47 @@ public void matchEnded(Match match) {
               if (manager.getOverriderMap() != null) return;
               // If there is a restart queued, don't start a vote
               if (RestartManager.isQueued()) return;
-              currentPoll = new MapPoll(match, mapScores, VOTE_SIZE);
+              currentPoll =
+                  shouldCustomVote()
+                      ? new MapPoll(
+                          match,
+                          getCustomVoteMapWeighted(),
+                          Math.min(MAX_VOTE_OPTIONS, customVoteMaps.size()))
+                      : new MapPoll(match, mapScores, VOTE_SIZE);
+
```

Comment:
I assume that custom votes should be added to the poll alongside the other maps, instead of completely overwrite

Reply:
@Pablete1234 When implementing, my idea was for a complete override. I feel this gives staff more control, so say if they only wanted players to select from 2 or 3 maps. Perhaps an additional flag could be added that will “fill” the remaining selections with random maps from the current pool?

Reply:
Yeah, how i understood it is if someone is nagging you about *HEY SN THIS MAP* you can just add it to the vote and that way players can decide if they wanna play it, alongside other 4 "normal" maps